Formula/sqitch.rb

```diff
@@ -0,0 +1,77 @@
+class Sqitch < Formula
+  desc       "Sensible database change management"
+  homepage   "https://sqitch.org/"
+  url        "https://www.cpan.org/authors/id/D/DW/DWHEELER/App-Sqitch-v1.3.1.tar.gz"
+  sha256     "f5e768d298cd4047ee2ae42319782e8c2cda312737bcbdbfaf580bd47efe8b94"
+  license    "MIT"
+  head       "https://github.com/sqitchers/sqitch.git", branch: "develop"
+
+  depends_on "cpm" => :build
+  depends_on "libiodbc"
+  depends_on "libpq"
+  depends_on "mysql-client"
+  depends_on "perl"
+  uses_from_macos "sqlite"
```

Comment:
I changed this from `depends_on` at the recommendation of `brew audit --new`, but as there is an [sqlite formula](https://formulae.brew.sh/formula/sqlite) that will install a newer version than on the OS, ISM it might be better kept `depends_on`.